### PR TITLE
cmd/snap: i18n option descriptions

### PR DESCRIPTION
**Pull request by chipaca**
[compare with prerequisite](https://github.com/chipaca/snappy/compare/freundlicher-snap-find...chipaca:i18n-option-descriptions).

### CODE REVIEW

File: cmd/snap/main.go [hunk 1]
```diff
@@ -68,12 +70,14 @@ var experimentalCommands []*cmdInfo
 
 // addCommand replaces parser.addCommand() in a way that is compatible with
 // re-constructing a pristine parser.
-func addCommand(name, shortHelp, longHelp string, builder func() flags.Commander) *cmdInfo {
+func addCommand(name, shortHelp, longHelp string, builder func() flags.Commander, optDescs map[string]string, argDescs [][2]string) *cmdInfo {
```
Comment (mvo5) [inline cmd/snap/main.go]: Maybe its just me, but I am not fond of `argDescs [][2]string`. Maybe `argDescs []Arg` with `struct Arg {Name, Descr string} ` Slightly more verbose but easier to read (I hope). WDYT?

Comment (chipaca) [inline cmd/snap/main.go]: I guess I'm too easily the wrong kind of lazy. You're right; I'll fix it.

File: cmd/snap/cmd_get.go [hunk 2]
```diff
@@ -35,15 +35,27 @@ The get command prints the configuration for the given snap.`)
...
+	addCommand("get", shortGetHelp, longGetHelp, func() flags.Commander { return &cmdGet{} },
+		map[string]string{
+			"d": i18n.G("always return document, even with single key"),
+		}, []argDesc{
+			{
+				name: i18n.G("<snap-name>"),
+				desc: i18n.G("the snap whose conf is being requested"),
+			},
+			{
+				name: i18n.G("keys"),
```
Comment (mvo5) [inline cmd/snap/cmd_get.go]: This is different from above, it used to be `<keys>` - not sure if that matters in any way though :)

File: cmd/snap/cmd_snap_op.go [hunk 3]
```diff
@@ -591,11 +617,19 @@ func (x *cmdRevert) Execute(args []string) error {
...
-	addCommand("install", shortInstallHelp, longInstallHelp, func() flags.Commander { return &cmdInstall{} })
-	addCommand("refresh", shortRefreshHelp, longRefreshHelp, func() flags.Commander { return &cmdRefresh{} })
-	addCommand("try", shortTryHelp, longTryHelp, func() flags.Commander { return &cmdTry{} })
-	addCommand("enable", shortEnableHelp, longEnableHelp, func() flags.Commander { return &cmdEnable{} })
-	addCommand("disable", shortDisableHelp, longDisableHelp, func() flags.Commander { return &cmdDisable{} })
-	addCommand("revert", shortRevertHelp, longRevertHelp, func() flags.Commander { return &cmdRevert{} })
+	addCommand("remove", shortRemoveHelp, longRemoveHelp, func() flags.Commander { return &cmdRemove{} },
+		map[string]string{"revision": i18n.G("Remove only the given revision")}, nil)
+	addCommand("install", shortInstallHelp, longInstallHelp, func() flags.Commander { return &cmdInstall{} },
+		channelDescs.also(modeDescs).also(map[string]string{
```
Comment (mvo5) [inline cmd/snap/cmd_snap_op.go]: Clever!

**Comment by mvo5** [comment]
Very nice! Thanks for doing this work.

**Pull request merged**